protocol.html

```diff
@@ -895,6 +895,8 @@ <h3 property="schema:name">Security Considerations</h3>
                   <p>Solid data pods <a href="#cors-server">disable all cross-origin protections</a> in browsers because resource access is governed explicitly by <a href="#web-access-control">Web Access Control</a>. As such, data pods MUST NOT rely on browser-based cross-origin protection mechanisms for determining the authentication status or representation of a resource. In particular, they MUST ignore HTTP cookies from untrusted origins. Additional security measures MAY be taken to prevent metadata in error responses from leaking. For instance, a malicious app could probe multiple servers to check whether the response status code is <code>401</code> or <code>403</code>, or could try to access an error page from an intranet server within the user agent’s private network to extract company names or other data. To mitigate this, when a request from an untrusted <code>Origin</code> arrives, the data pod MAY set the status code of error responses to <code>404</code> and/or anonymize or censor their contents.</p>
 
                   <p>Data pods SHOULD use TLS connections to protect the contents of requests and responses from eavesdropping and modification by third parties. Unsecured TCP connections without TLS MAY be used in testing environments or when the data pod is behind a reverse proxy that terminates a secure connection.</p>
+
+                  <p>When a server responds to an authorized HTTP <code>GET</code> request, the response MUST NOT expose information about resources that the agent is unauthorized to read. For example, when a <code>GET</code> method request targets a container, the server MUST NOT include information besides containment statements about the contained resources in the response eg. last modification, size, type or label.</p>
```

Comment:
```suggestion
                  <p>When a server responds to an authorized HTTP <code>GET</code> request, the response MUST NOT expose information about resources that the agent is unauthorized to read. For example, when a <code>GET</code> method request targets a container, the server MUST NOT include information beyond containment statements about the contained resources in the response. Last modification time, size, type or label are examples of what is not allowed in that scenario.</p>
```

Looks good - suggestion is aimed to ensure people don't misread and lump those examples (mod time, etc) in the "besides these" group.

Reply:
Makes sense, thanks.